metadata-ingestion/src/datahub/ingestion/source/ge_data_profiler.py

```diff
@@ -305,8 +305,15 @@ def _get_column_type(self, column_spec: _SingleColumnSpec, column: str) -> None:
     def _get_column_cardinality(
         self, column_spec: _SingleColumnSpec, column: str
     ) -> None:
-        nonnull_count = self.dataset.get_column_nonnull_count(column)
-        column_spec.nonnull_count = nonnull_count
+        try:
+            nonnull_count = self.dataset.get_column_nonnull_count(column)
+            column_spec.nonnull_count = nonnull_count
+        except Exception as e:
+            logger.debug(f"Exception was {e}")
```

Comment:
How about: 
"Caught exception while attempting to get column cardinality for column {column}. {e}"